contracts/wveOLAS.sol

```diff
@@ -89,6 +128,20 @@ contract wveOLAS {
         ve = _ve;
     }
 
+    /// @dev Gets the most recently recorded user point for `account`.
+    /// @param account Account address.
+    /// @return pv Last checkpoint.
+    function getLastUserPoint(address account) external view returns (PointVoting memory pv) {
+        pv = IVEOLAS(ve).getLastUserPoint(account);
+    }
+
+    /// @dev Gets the number of user points.
+    /// @param account Account address.
+    /// @return userNumPoints Number of user points.
+    function getNumUserPoints(address account) external view returns (uint256 userNumPoints) {
+        userNumPoints = IVEOLAS(ve).getNumUserPoints(account);
+    }
+
```

Comment:
why the next one is public and not external?

Reply:
typo, fixed